.gitignore

```diff
@@ -1 +1,2 @@
 *.test
+.idea
```

Comment:
Please get rid of this, I don't think it belongs here. If you need to ignore `.idea` files, you can add it to your personal `.git/info/exclude` file, which doesn't get committed. See https://help.github.com/articles/ignoring-files/#explicit-repository-excludes.